xml_inscriptions/transcribed/CA.Berk.UC.HMA.L.8-2330.xml

```diff
@@ -124,6 +126,13 @@
    <text>
       <body>
          <div type="edition">
+            
```

Comment:
This needs to be a separate div, I think -- effectively, what you want to do hierarchically (if you're going to include both) is to have a separate `<div type="textpart" subtype="XXX">` with the subtypes being "diplomatic" and "interpretive" (or "editorial"); as it stands, the semantic markup will understand the edition you produced to be four lines of text that all appear on the object.